package di.aittr.pro_sotrudnikov.exeption_handling.handling.exeptions;

public class SotrudnikValidacionExeption extends RuntimeException{

    public SotrudnikValidacionExeption(Throwable cause) {
        super(cause);
    }
}
